client/my-sites/plugins/plugin-management-v2/plugin-details-v2/style.scss

```diff
@@ -162,4 +162,13 @@
 			justify-content: flex-end;
 		}
 	}
+// Hiding this here because in this version the mediaField is not hiddable
+// https://github.com/WordPress/gutenberg/tree/d59faffa3d270dbc7716f653b4cdab6020812658/packages/dataviews#properties-of-layout
+// In future versions, a showMedia property can be used
+	.dataviews-view-list {
+		.dataviews-view-list__media-wrapper {
+			background-color: transparent;
+			display: none;
```

Comment:
This can be revisited once the latest version of the `@wordpress/dataviews` component is deployed as part of https://github.com/Automattic/wp-calypso/pull/96470 because in the latest version there is a `showMedia` property: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dataviews/#properties